frontend/src/app/main/data/workspace/libraries.cljs

```diff
@@ -944,7 +944,20 @@
                                  (when sync-typographies?
                                    (dwlh/generate-sync-file it file-id :typographies asset-id library-id state))])
 
-               changes         (pcb/concat-changes library-changes file-changes)]
+               changes         (pcb/concat-changes library-changes file-changes)
+
+               find-shape      (fn [data]
```

Comment:
We should add a rule that forbids naming a parameter `data` :grimacing:.

When I read this code, i have absolutely no clue of what kind of data comes here... If this is a change, the name should be `change`. When we have not data types, at least the name helps us to understand what data are we talking about.

Reply:
Also the name `find-shape` suggests that it returns one shape, but it actually returns a list, don't it?

Reply:
Fixed